### DISCUSSION TITLE
Updating a DAG's schedule interval

### DISCUSSION THREAD

**Discussion by SergioJFalcon** [apache/airflow#31903] [Q&A]
I'm having trouble updating an existing deployed DAG's schedule interval, is there an API endpoint from AirFlow that will allow me to update the schedule interval? As far as I can see, the DAG update endpoint only allows me to update the is_paused property.

**Comment by potiuk** [comment]
no. It comes exclusively from DAG definition (i.e file) - but you can use any API that your "dag distribution" mechanism provides  - write a new copy of the DAG with changed schedule. This is also an API usually that can be used via automation (git, S3, gcs etc.).
